File: frontend/android/app/src/main/java/com/example/hifes/data/model/MarkerDto.kt
package com.example.hifes.data.model

data class MarkerDto(
    var markerId: Int,
    var festivalId: Int,
    var hostId: Int,
    var boothName: String,
    var boothLatitude: Float,
    var boothLongitude: Float,
    var showingImg: String,
    var description: String,
    var boothNo: Int,
    var markerColor: String
)

File: frontend/web/lib/festival/MarkerDto.dart
class MarkerDto {

  final String boothName;
  final String description;
  final int boothNo;
  final double boothLatitude;
  final double boothLongitude;


  MarkerDto({
    required this.boothName,
    required this.description,
    required this.boothNo,
    required this.boothLatitude,
    required this.boothLongitude,
  });

  Map<String, dynamic> toJson() {
    return {
      'boothName': boothName,
      'description': description,
      'boothNo': boothNo,
      'boothLatitude': boothLatitude,
      'boothLongitude': boothLongitude,
    };
  }

  @override
  String toString() {
    return 'MarkerDto{'
        'boothName: $boothName, '
        'description: $description, '
        'boothNo: $boothNo, '
        'boothLatitude: $boothLatitude, '
        'boothLongitude: $boothLongitude, }';
  }
}
